webserver/src/main/java/io/kestra/webserver/controllers/FlowController.java

```diff
@@ -198,6 +214,14 @@ public PagedResults<Flow> find(
         ));
     }
 
+    @ExecuteOn(TaskExecutors.IO)
+    @Get(uri = "/{namespace}", produces = MediaType.TEXT_JSON)
+    @Operation(tags = {"Flows"}, summary = "Retrieve all flow id from a given namespace")
+    public List<String> getFlowsIDByNamespace(
```

Comment:
Why map only `flowId`, it's not restful

Reply:
We thought it would save some data but good point I'll revert back to full flows

Reply:
In fact since it's not paginated I was wondering about the amount of data we transfer but idk if you think it's okay like that why not, just know that there is no limit and for large clusters it may be a bit long :thinking: